core/src/smtp/mod.rs

```diff
@@ -198,26 +198,23 @@ async fn email_deliverable(
 		.command(RcptCommand::new(to_email.clone(), vec![]))
 		.await
 	{
-		Ok(response) => match response.first_line() {
-			Some(message) => {
-				let message = message.to_lowercase();
-				let is_deliverable = message.contains("2.1.5") || 
-					// 250 Recipient address accepted
-					// 250 Accepted
-					message.contains("accepted")||
-					// 250 OK
-					// 250 Recipient <email> OK
-					message.contains("ok");
-				Ok(Deliverability {
-					has_full_inbox: false,
-					is_deliverable,
-					is_disabled: false,
-				})
-			}
-			None => Err(SmtpError::SmtpError(AsyncSmtpError::Client(
-				"No response on RCPT command",
-			))),
-		},
+		Ok(_) => {
+			// According to RFC 5321, `RCPT TO` command succeeds only with
+			// 250 and 251 codes: https://tools.ietf.org/html/rfc5321#page-56
+			//
+			// Where the 251 code is used for forwarding, which is not our case,
+			// because we always deliver to the SMTP server hosting the address
+			// itself.
+			//
+			// So, if `response.is_positive()` (which is a condition for
```

Comment:
It seems to me that `is_positive()` is `2xx | 3xx`:

https://github.com/async-email/async-smtp/blob/2275fd8d13e39b2c58d6605c786ff06ff9e05708/src/smtp/response.rs#L167-L173

Could you add a comment saying that 3xx responses are not possible in `RCPT TO`

Reply:
It's described in the RFC link, I've referred in the comment. But, of course, I can refer explicitly, in addition.

Reply:
@amaurymartiny done